isamples_metadata/isamplesfasttext.py

```diff
@@ -0,0 +1,44 @@
+# Adapted from https://github.com/isamplesorg/vocabulary_learning/blob/develop/fasttext_interface/DwC_predict.py
+
+import fasttext
+import os.path
+import typing
+import re
+
+from fasttext.FastText import _FastText
+
+MODEL_FILENAME = "./sampledFeature.bin"
+THIS_PATH = os.path.dirname(os.path.abspath(__file__))
+MODEL_PATH = os.path.join(THIS_PATH, MODEL_FILENAME)
+MODEL = None
+RESULT_PREFIX = "__label__"
+
+
+def _model() -> _FastText:
```

Comment:
@datadavev this isn't thread-safe as written.  Given our current usage I think it's probably ok, but I was wondering if you have a pointer at a python idiomatic way of handling something like this.

Reply:
Perhaps create a class that wraps the fastText transformer to include the configuration properties as parameters. Then in FastAPI app, create an instance of the class during startup and shared as needed